frontend/choose-lang.php

```diff
@@ -298,15 +298,30 @@ public function parse_main_query( $query ) {
 			$this->set_language( $lang );
 			$this->set_curlang_in_query( $query );
 		} elseif ( ( count( $query->query ) == 1 || ( is_paged() && count( $query->query ) == 2 ) ) && $lang = get_query_var( 'lang' ) ) {
```

Comment:
Could we use strict comparison? And also, is it the right time to get rid of variable assignation in `if`?